### ISSUE TITLE
No disconnection events

### ISSUE THREAD

**Issue by Riberis** [AuspeXeu/openvpn-status#34]
Hi,

I have followed the intructions from here https://www.npmjs.com/package/openvpn-status and the gui works but it does not display disconnection events. 

Is there any status.sh given? The one i found was in another issue https://github.com/AuspeXeu/openvpn-status/issues/11.

**Comment by AuspeXeu** [comment]
Yeah it seems like this has been introduced with the latest update.

**Comment by Riberis** [comment]
So any idea how to resolve it?
